db/migrate/20150416231106_add_source_to_webhook.rb

```diff
@@ -0,0 +1,12 @@
+class AddSourceToWebhook < ActiveRecord::Migration
+  def up
+    add_column :webhooks, :source, :string, null: false, default: 'any_ci'
+    change_column_default :webhooks, :source, nil
```

Comment:
@grosser @steved changed to make it `null false` and not only use the default value to update old records, verified that the default is applied to existing rows in all 3 dbs.
I used up and down because this isn't reversible otherwise.
The other option would be to go `Webhook.update_all source: 'any_ci'`